com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -227,6 +228,23 @@ public ulong LocalClientId
 
         internal static event Action OnSingletonReady;
 
+        // the interest settings objects receive unless they have a pre-prefab override
+        public InterestSettings InterestSettings;
+
+        private InterestManager m_InterestManager;
```

Comment:
We have a bunch of other "managers" above that are managed the same way (all assigned in Init() and destroyed in Shutdown())
Any reason why this is different here?